src/features/asciidoctorConfig.ts

```diff
@@ -17,9 +17,28 @@ export async function getAsciidoctorConfigContent (documentUri: vscode.Uri): Pro
   if (workspaceFolder === undefined) {
     return undefined
   }
-  const asciidoctorConfigUri = vscode.Uri.joinPath(workspaceFolder.uri, '.asciidoctorconfig')
+  const configContentFromAsciidoctorConfig = await getConfigContent(workspaceFolder, '.asciidoctorconfig')
+  const configContentFromAsciidoctorConfigDotAdoc = await getConfigContent(workspaceFolder, '.asciidoctorconfig.adoc')
+  if (configContentFromAsciidoctorConfig !== undefined) {
+    if (configContentFromAsciidoctorConfigDotAdoc !== undefined) {
+      return configContentFromAsciidoctorConfig + configContentFromAsciidoctorConfigDotAdoc
```

Comment:
The Intellij IDEA plugin joins configuration files using two newlines:

https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/64707c7632e1c5e6efc34100dc3cb2dd0bab4ea6/src/main/java/org/asciidoc/intellij/AsciiDocWrapper.java#L727

Reply:
sounds overkill as we already have two newlines at end of the grabbed content but well, better to stay as close as possible to IntelliJ